### DISCUSSION TITLE
nebulizzazione antizanzare

### DISCUSSION THREAD

**Discussion by Nik71git** [indomus/forum#2284] [Nuove idee, proposte, progetti e guide]
Come anticipato in chat apro questa discussione per elaborare un progetto per la nebulizzazione automatizzata, l'esigenza nasce per eliminare le zanzare ma sarà applicabile a qualsiasi esigenza di nebulizzazione. La mia idea è di utilizzare una board ESP con Esphome.
Chiunque voglia proporre idee e/o materiali si aggiunga pure

**Comment by maurizioandreotti60** [comment]
Seguo perché mi interessa

**Comment by Nik71git** [comment]
Prima di cominciare chiariamoci un po' le idee sulla parte meno domotica.
Di seguito un link ad un interessante blog che tratta l'argomento. Ce ne sono altri in rete ma evitiamo di pubblicare qui roba che contiene parti e-commerce.

https://www.fresconaturale.com/blog/nebulizzazione-antizanzare-fai-da-te/

**Reply by Nik71git** [reply]
il progetto sta prendendo forma, paradossalmente la parte più difficile è quella meno domotica quindi qualsiasi suggerimento è bene accetto sul dimensionamento della pompa (portata e pressione).
L'idea è quella di sfruttare un Sonoff 4CH pro che piloti due zone da 10/15 ugelli ciascuna (20 metri per zona) con due elettrovalvole, un terzo canale per pilotare la pompa ed il quarto per il lavaggio con sola acqua.
Gli sprinkler di esphome con la loro gestione estremamente flessibile sono l'ideale.

**Comment by antonellopasella** [comment]
Io prevederei un modo per dire quanti ugelli sono connessi a ogni linea e da quanti l/h sono, in modo da fare un conto a spanne del consumo (per eventuali cisterne)

idem per un sensore temperatura in caso di repellenti termo sensibili (inutile spruzzare se temperatura troppo alta in alcuni casi)

magari anche un sensore pioggia (inutile spruzzare se piove)

**Reply by Nik71git** [reply]
per temperatura o pioggia uso i dati della stazione meteo fatta sempre con esphome

**Comment by antonellopasella** [comment]
Sapendo numero di ugelli e portata si può risalire alla portata necessaria alla pompa per il dimensionamento (anche qui, warning se programmi più linee di quelle che puoi sopportare)

Per la pressione occorre dimensionarla in base alla lunghezza massima della tratta.

Anche qui, possiamo mettere qualcosa in configurazione, oppure dare delle indicazioni di massima per la scelta.

**Comment by Nik71git** [comment]
domani mi arriva tutto il materiale mancante e poi parto con l'assemblaggio; purtroppo la prima versione sarà molto basica, una sola linea, niente elettrovalvole e niente sensore di livello liquido. il tutto collegato ad una presa già integrata in HA con Esphome.
mi riservo in futuro di implementare meglio il tutto ma ora devo seguire altre cose ed ho la necessità di far partire anche questa.

**Comment by Ocramius** [comment]
Volevo solo sapere se il progetto fosse andato da qualche parte: gli impianti commercializzati sono folli (1500€ + installazione, e ultimamente anche subscription), il tutto per un circuito idraulico con una pompa peristaltica/miscelatore, quindi sarebbe interessante il fai-da-te 😁

**Reply by Nik71git** [reply]
il progetto è stato abortito per mancanza di hardware adeguato, le prove al banco sono andate bene ma serve una pompa di buona portata e un serbatoio di almeno 150/200 litri; l'implementazione con esphome di per sé non è difficile (simile all'irrigazione) ma per il momento proseguo con il classico spruzzatore manuale.

**Reply by Ocramius** [reply]
Grazie per l'update :-)

**Comment by Gagelefa** [comment]
Ciao a tutti, se a qualcuno interessa sono riuscito, dopo svariate ricerche e tentativi, nel mettere assieme un hardware adeguato. Ho costruito un nebulizzatore in grado di supportare oltre 100 metri di tubo. Pesca in automatico da due prodotti differenti e l'ho collegato direttamente all'acquedotto. Al momento è collegato in home assistant in modalità molto basic ma mi piacerebbe sviluppare la parte di integrazione. Se qualcuno è interessato parliamone. Ciao

**Reply by Nik71git** [reply]
per quanto riguarda la parte hardware puoi postare il tuo setup?
per la parte di codice io avrei voluto usare esphome e comandare i nebulizzatori come fossero degli sprinkler in quanto la gestione delle valvole è piuttosto completa

**Reply by Ocramius** [reply]
Decisamente interessato a vedere almeno una foto dell'hardware: sono molto convinto che il [mio setup](https://github.com/indomus/forum/discussions/2284#discussioncomment-9929917) sia molto overkill :D

**Reply by Gagelefa** [reply]
Ciao a tutti, per i prossimi 15 giorni sono via per lavoro ma appena rientro condivido, magari con delle foto, il progetto. Ho speso circa 1000 euro di hardware tra pompe, peristaltiche, tubi, raccordi, ugelli e impianto elettrico. Alla fine ho collegato il tutto in home assistant ma da quel punto di vista ho necessità di ottimizzare in quanto non ho nozioni approfondite di programmazione.

**Reply by skywalker1979** [reply]
Ciao @Gagelefa
a me interessa e molto....sto cercando di fare la stessa cosa e sarei interessato a capire come misceli la soluzione....
Io sto provando con un collettore in alluminio collegato a delle pompe peristaltiche....ma non so se possa essere la soluzione giusta....
tu come effettui la miscelazione?
Ho un pozzo da cui prenderei l'acqua, già collegato a una pompa che mi sembra più che potente per quello che devo fare....quindi non metterei ulteriori pompe a meno che non sia necessario per la miscelazione.
Ho pensato ingresso acqua -> collettore alluminio con ulteriori ingressi per la miscelazione -> uscita dal collettore, ingresso nel tubo di nebulizzazione....
Gli altri "Ingressi" del collettore, collegati a una valvola di non ritorno permetterebbero solo l'ingresso dei vari insetticidi tramite la/le pompa/e peristaltica
La mia idea, è di avere una board con un nodemcu. Ho già una bozza di firmware pronto che permette tramite una chiamata http di attivare o disattivare uno o più relè.....il resto lo si gestisce dalla domotica.
Ho già disegnato la pcb alcuni anni fa per altri scopi e l'ho fatta creare in cina a pochi euro....ma la parte software / elettronica è la parte più semplice.
La parte meno intuitiva è quella idraulica.
spesa totale fino a ora circa 150€ perchè ho comprato su amazon anche cose che forse non mi serviranno. Sono convinto che in negozio si riesca a spendere la metà....altrochè quelli commerciali. 
Mi manca una specie di contenitore....e probabilmente per quest'anno se faccio in tempo a montarlo finirà su un pannello di legno....

**Reply by skywalker1979** [reply]
mi è venuto però un dubbio.
con quanto fatto, cioè collettore, con valvole di non ritorno etc. ho il grande timore che se la pressione dell'acqua dentro il collettore è alta, come mi aspetto, le pompe peristaltiche non abbiano la forza di aprire la valvola di non ritorno per iniettare la soluzione nel collettore.....e ho paura che renderebbe necessario aggiungere una ulteriore pompa a valle del collettore.....che non vorrei ....

**Reply by Gagelefa** [reply]
Eccomi qua, scusa ma è un mese piuttosto intenso. Il tuo progetto è leggermente diverso da quello che ho fatto io ma poco cambia. Tieni presente che normalmente, una valvola di non ritorno (ad esempio con attacco da 1/8") con un tubo di ingresso di 6 mm, ha una resistenza di circa 1 bar, pertanto la tua o le tue peristaltiche devono avere una potenza pari o superiore ad 1 bar. Per quanto riguarda la miscelazione la faccio fare direttamente alla pompa, a monte ho l'ingresso dell'acqua e un raccordo a T con l'innesto della valvola di non ritorno collegata alla peristaltica. Tieni presente, da miei esperimenti, che se la pompa non supera i 15 bar l'effetto del prodotto nebulizzato resiste ben poco nell'aria; potresti "semplicemente" risolvere aumentando le erogazioni di prodotto durante la giornata.

**Reply by skywalker1979** [reply]
l'hai già messo in azione quindi? e hai risolto con quei mostri volanti o ne hai ancora che svolazzano?
Se ti va e non ti scoccia pubblica una foto che sono molto interessato!
Hai risolto con la parte elettronica?

**Reply by Gagelefa** [reply]
Il nebulizzatore è operativo e pesca da due differenti prodotti per un totale di 5 volte al giorno. Di sera erogo l'abbattente e di giorno un disabituante. Al momento non sono ancora rientrato in Italia ma appena riesco condivido una foto, nessun problema. Per quanto riguarda la parte elettronica gestisco tutto con Home assistant e per quanto riguarda il livello dei prodotti con dei galleggianti e due ESP32 anch'esse gestite da Home Assistant. Ho aggiunto inoltre una sonda tuya che mi rileva i consumi dell'acqua e l'eventuale rottura o uscita anomala di acqua dal sistema, ho aggiunto a monte una elettrovalvola che chiude tutto il circuito (sempre gestita da home assistant).

**Reply by Gagelefa** [reply]
Ps. problema zanzare tigre totalmente risolto

**Reply by skywalker1979** [reply]
> Ps. problema zanzare tigre totalmente risolto

davvero? ma questa è una notizia bellissima.... non hai più zanzare tigre in giardino? devo accelerare con l'installazione....! da me non posso neanche uscire di casa senza che mi assalgano.....!!
e le zanzare normali? perchè anche di quelle io ne ho un belpò!!

**Reply by skywalker1979** [reply]
> Il nebulizzatore è operativo e pesca da due differenti prodotti per un totale di 5 volte al giorno. Di sera erogo l'abbattente e di giorno un disabituante. Al momento non sono ancora rientrato in Italia ma appena riesco condivido una foto, nessun problema. Per quanto riguarda la parte elettronica gestisco tutto con Home assistant e per quanto riguarda il livello dei prodotti con dei galleggianti e due ESP32 anch'esse gestite da Home Assistant. Ho aggiunto inoltre una sonda tuya che mi rileva i consumi dell'acqua e l'eventuale rottura o uscita anomala di acqua dal sistema, ho aggiunto a monte una elettrovalvola che chiude tutto il circuito (sempre gestita da home assistant).

io invece ho intenzione di usare la board di cui ti ho scritto qualche messaggio fa che ha supporto per 5 o 6 relè su una unica board....così mi basta attivare i relè sulla board in successione e dalla domotica faccio una chiamata sola....quando avrò il progetto per arduino lo posto qui cmq....e se interessa anche il disegno della board.

**Reply by Gagelefa** [reply]
Ciao, forse mi sono un pezzo ma non capisco il perché utilizzi la nodemcu quando potresti usare degli switch tuya che colleghi in homeassistant, dove creerai delle automazioni....

**Reply by skywalker1979** [reply]
boh...per quello sono preferenze....
io preferisco avere un pulsante unico nella domotica che dice "attiva" a una certa ora....e il resto viene gestito dal node per esempio tempi, quali relè attivare etc.
probabilmente si fa la stessa cosa con i tuya ma serve più integrazione nella domotica....

**Reply by Gagelefa** [reply]
Effettivamente hai ragione ma probabilmente, in passato, sono stato sfortunato io in quanto non ho mai trovato stabilità hardware e software con arduino! Da quando utilizzo homeassistant e non ho più usato arduino ho risolto parecchi problemi di stabilità ma non faccio testo :)

**Reply by skywalker1979** [reply]
l'importante è il risultato..! e se hai fatto sparire le zanzare dal tuo giardino hai vinto....non importa come! 🤣

**Comment by Ocramius** [comment]
Nel frattempo io ho costruito qualcosa di rudimentale, che non ho ancora collaudato (purtroppo Maggio/Giugno sono stati spietati).

Il mio motivo principale per non comprare un prodotto off-the-shelf e' che ho anche arnie/api, e quindi non posso assolutamente permettere la propagazione di insetticidi che non conosco nel dettaglio.

La mia idea di fondo:

 1. tanica con galleggiante accumula acqua di acquedotto a pressione ambiente
 2. pompa peristaltica sposta acqua in un miscelatore secondario
 3. feeder per acquario fa cadere antizanzare BTI ("Bacillus Thuringiensis Israelensis") nel miscelatore. Notare che il BTI e' fotosensibile / va tenuto al fresco e al buio. Devo ancora trovare un feeder adatto.
 4. pompa peristaltica reimmette liquido miscelato nella tanica
 5. pompa ad alta pressione indirizza la miscela verso gli ugelli nebulizzatori

Posto un paio di foto per dare un'idea del setup:

![raw setup on a test frame](https://github.com/indomus/forum/assets/154256/78e98ced-6f7a-4cf6-8940-567269c647cf)
![setup on the final plate for installation](https://github.com/indomus/forum/assets/154256/592b1936-a95a-42ce-824a-18090d4b492d)

Ed ecco anche il setup in azione: https://photos.app.goo.gl/zgqvFdMryEohs4eQ7

**Reply by Ocramius** [reply]
Appena monto la cassetta all'impianto idraulico di casa (devo ancora mettere un riduttore di pressione) vedo di postare altre foto a riguardo, se c'e' interesse.

Notare che i costi non sono per nulla piccoli: ho speso molto, ma decisamente meno di meta' che un sistema ZanZero o simile.

Ad utenti piu' "casual", visto che si tratta di impianti con acqua ed elettricita' coinvolti, consiglio di seguire prima i canali commerciali :-)

**Reply by Gagelefa** [reply]
Ciao, come primo esperimento ho anche io utilizzato una tanica ma alla fine ho optato per miscelare acqua e prodotto direttamente in mandata, questo mi permette maggiori regolazioni e più potenza di erogazione senza passare da nessuna tanica. Ho capito, prima di arrivare a questa ultima versione, che la potenza di erogazione (BAR) è direttamente proporzionale all'efficacia contro gli insetti, ovvero; se la portata della pompa è al di sotto dei 20 BAR le particelle di prodotto e acqua che escono dagli ugelli (nebulizzazione) si depositano sul terreno più in fretta e pertanto l'effetto contro le zanzare è sensibilmente più basso, maggiore è la pressione e maggiore è la volatilità del prodotto che rimane "sospeso" in aria per più tempo. In sostante, meno erogazioni, meno prodotto = meno zanzare! C'ho sbattuto la testa e soldi, non pochi, prima di arrivarci!

**Reply by Gagelefa** [reply]
La pompa che stai utilizzando, se non erro, eroga circa 6 / 7 bar e probabilmente avrà anche un decadimento in quanto non hai la spinta dell'acquedotto che normalmente è tra i 2 / 3 Bar. Mi rendo conto di essere "malato" di dettagli ma a casa mia, con le zanzare tigre, ho dovuto per forza fare un sistema ad-hoc..

**Reply by Ocramius** [reply]
Hai per caso una pompa di riferimento?

Avevo decisamente i miei dubbi sulla pompa, ma sarei intanto contento di mandare i primi ~30m di tubo con ugelli dove so che ci sono regolari nidiate di zanzare.

**Reply by Gagelefa** [reply]
Ciao, è una pompa a palette industriale. Per i dettagli sentiamoci in privato. thx

**Reply by gigettino** [reply]
ciao a tutti, vorrei anch'io realizzare un sistema antizanzare a nebulizzazione.
Ho letto molte difficoltà e che forse la realizzazione non è così semplice ed economico.

A che punto siete ?
Se avete già del materiale da studiare sono qui.
grazie
Max

**Reply by Marco46984** [reply]
> Ciao, è una pompa a palette industriale. Per i dettagli sentiamoci in privato. thx

Buonasera ho seguito l'intero post con interesse e sto realizzando un impianto antizanzare a nebulizzazione, se non sono indiscreto vorrei sapere che tipo di pompa ha utilizzato per avere 20 bar di pressione.
La ringrazio in anticipo
Saluti

**Reply by Gagelefa** [reply]
@Marco46984 ho utilizzato una pompa a palette dalla Sisme Spa.

**Reply by Marco46984** [reply]
Potrei sapere le caratteristiche tecniche perché sono giorni che cerco online ma è una giungla e quindi vorrei prendere una pompa più adeguata possibile considerando che svilupperó un impianto di circa 120/140 mt con 50 ugelli su uno stesso livello e probabilmente a 2 settori.
Grazie mille

**Comment by marta312** [comment]
@Gagelefa sarei interessata a discutere con te l'hardware che hai usato.. potresti darmi qualche informazione sulla pompa e sugli ugelli? Stavo valutando pomoe da 8bar ma a quanto dici potrebbero non essere sufficienti... Grazie!

**Reply by gigettino** [reply]
ciao , dipende molto da che tratta devi coprire. Io mi sto orientando sui 20bar perché devo fare 60/70m con 3 linee.
tu come ti stai organizzando ?

**Reply by Gagelefa** [reply]
@marta312 Scusa se rispondo solo ora ma non mi arrivano le notifiche!!! Il sistema non è complesso ma ci sono parecchi items, la pompa è una pompa industriale a palette da 20 BAR, ce ne sono di tanti brand io ho scelto quella della Sisme.

**Reply by marta312** [reply]
@Gagelefa  Grazie mille! Hai preso una pompa già con il motore?

**Comment by marta312** [comment]
Ciao, grazie della risposta!
Anche io pensavo di dividere su un paio di linee... devo girare tutta la
casa e in totale saranno più di 150 metri lineari da coprire con tubo e
ugelli.
Però probabilmente con 20 bar si dovrebbe riuscire anche con una sola linea.

Ho studiato un po di sistemi commerciali. Stocker (il più economico) sembra
lavorare a 5 bar, ma suggerisce di fare 2/3 linee o una linea ma chiusa...

Quello che mi piace di più è quello di freezanz con le pompe peristaltiche:

https://www.zanzamaipiu.it/gli-impianti-antizanzare-a-nebulizzazione-sono-tutti-uguali/

Loro lavorano a 20bar, ma fanno impianti grossi.

Non volevo montare la pompa in linea con l'acquedotto per non rischiare
contaminazioni in caso di ritorno... so che ci sono anche delle regole
stringenti su questo.

Al momento non sono riuscita a trovare una pompa a 20bar... ce ne sono un
sacco, anche economiche usate per i camper ma al massimo arrivano a 8bar...
in più c'è da capire se sono robuste alla miscela che vi circola... se non
sbaglio, avevo letto che suggerivano quelle a membrana proprio per questo...

Se hai suggerimenti o esperienze a riguardo, sono tutti bene accetti!


Il Gio 5 Set 2024, 18:02 gigettino ***@***.***> ha scritto:

**Reply by gigettino** [reply]
io devo fare per forza 3 linee e non riesco a fare l'anello chiuso.
Le peristaltiche le devi usare per forza se vuoi mixare i liquidi
per il ritorno basta usare una valvola di non ritorno
appena ho idee + chiare magari ci condividiamo il materiale.

**Reply by Gagelefa** [reply]
@gigettino @marta312 se posso dare un consiglio è quello di utilizzare pompe che abbiano una pressione superiore ai 15 BAR in quanto il prodotto deve essere nebulizzato nel modo corretto, altrimenti si rischierebbe che le particelle espulse non rimangano sospese sufficientemente nell'aria depositandosi immediatamente sul terreno, con eccessivo utilizzo di prodotti senza ottenere il risultato sperato. 

Per la contaminazione dell'acqua concordo con la valvola di non ritorno e a monte un filtro dell'acqua.

**Comment by marta312** [comment]
Si, anche io farò qualche conto più preciso e se ho novità ti scrivo!
Gli ugelli l'hai già selezionati?

Il Gio 5 Set 2024, 18:38 gigettino ***@***.***> ha scritto:

**Comment by marta312** [comment]
Ciao,
Facendo un conto veloce con 20bar e 50 ugelli da 0.1mm a me viene fuori
circa 1.3L/min, con una portata per ugello di circa 0.027L/min.
Può tornare secondo te?

Il problema è che non riesco a trovare un'elettropompa adeguata, già con
pompa e motore.

Tu hai indicazioni a riguardo? Se non sbaglio nel forum parlavi di una
pompa industriale a palette...

Il Gio 5 Set 2024, 18:49 Marta Niccolini ***@***.***> ha
scritto:

**Reply by gigettino** [reply]
Ciao, sto facendo fare il dimensionamento ad un esperto che sarà anche il fornitore di tutto il materiale (è un rivenditore). Lui mi darà tutta la parte idraulica io ci metto l'elettronica.
Avevo l'ugello da 0,4 o 0,5, tu come calcoli ?

**Reply by Gagelefa** [reply]
@gigettino @marta312 "purtroppo" il calcolo non è così automatico in quanto qui entriamo nel campo della fluidodinamica e in termini tecnici, sono c@@i :).

Bisogna calcolare la lunghezza del tubo oltre che il diametro, i BAR e i litri minuti erogati dalla pompa, oltre che il numero di ugelli e di conseguenza il diametro dell'ugello singolo. La cosa più "semplice" da fare è quella di installare un flussostato in uscita dell'impianto, a quel punto puoi fare il calcolo corretto delle peristaltiche!

**Reply by marta312** [reply]
@Gagelefa grazie del commento. La mia idea è quella di premiscelare in un serbatoio e poi usare una pompa autoadescante. In questa condizione credo che i calcoli fatti siano una approssimazione abbastanza corretta (validata su un software di simulazione)

La tua idea è quella di mettere la pompa in linea senza serbatoio, giusto? In questo caso concordo con te che i conti fatti sono troppo approssimativi per tarare le peristaltiche... a me servono per dimensionare la pompa principale autoadescante infatti.

**Reply by Gagelefa** [reply]
@marta312  Buongiorno, scusa ma non mi collegavo al forum da mesi!!! Confermo che ho collegato le pompe peristaltiche direttamente alla pompa a palette.

**Reply by superdani884** [reply]
@Gagelefa  buongiorno, sto provando anche io a costruire un impianto a nebulizzazione ad alta pressione. Volevo chiedere se puoi gentilmente condividere qualche altro dettaglio tecnico sul tuo progetto. Leggendo i tuoi post precedenti fai riferimento a una pompa Sisme da 20 bar ma Sisme mi risulta che fa solo motori e non pompe, cercando sempre sul web ho trovato pompe per impianti a osmosi della fluid o tech da 20 bar massimi teorici, chissà se però va bene...

**Comment by marta312** [comment]
Credo che, visto il poco flusso e le piccole perdite, si possa considerare
costante la pressione su ogni ugello.
Quindi si può avere una stima della velocità di uscita e della massa in
uscita con le formule che ti allego.

Il diametro dell'ugello è parametrico... al momento non so bene come
sceglierlo.. ho paura che 0.1mm faccia il getto troppo stretto..

Sicuramente non saranno precise al centesimo ma secondo me è una buona
stima di partenza.

![20240905_224508](https://github.com/user-attachments/assets/89ba30d2-9128-4843-9827-85155aa89f41)



Il Sab 7 Set 2024, 13:33 gigettino ***@***.***> ha scritto:

**Reply by gigettino** [reply]
ciao, non vedo l'allegato :-(

**Reply by gigettino** [reply]
ok lo vedo, poi lo guardo grazie 1000

**Comment by marta312** [comment]
Il Lun 9 Set 2024, 09:18 gigettino ***@***.***> ha scritto:

**Comment by gigettino** [comment]
scusami tanto ma continuo a non vedere gli allegati.. :-(

**Comment by marta312** [comment]
Ma davvero? Neanche la foto embedded nel testo della mail?

Il Lun 9 Set 2024, 16:00 gigettino ***@***.***> ha scritto:

**Reply by Ocramius** [reply]
@marta312 le risposte via email non arrivano sul thread su Github: caricale da interfaccia web da https://github.com/indomus/forum/discussions/2284#discussioncomment-10593543

**Reply by marta312** [reply]
Fatto, dimmi se la vedi!
